docs/README.md

```diff
@@ -26,16 +26,21 @@ The tests currently cover
 - Advection of a smoothed sphere at an angle on a *static* grid, on a *static* grid a twice the resolution, and with *AMR* covering the sphere at the effective higher resolution
 - Advection of a sharp sphere at an angle with *AMR* writing hdf5 output and comparing against a gold standard output.
 
-To execute the tests first obtain the current gold standard output
-```bash
-# from within the main parthenon directory
-wget -qO- https://pgrete.de/dl/parthenon_regression_gold_latest.tgz | tar -xz -C tst/regression/gold_standard
-```
-and afterwards run the tests, e.g., through
+To execute the tests run, e.g.,
 ```bash
 # from within the build directory (add -V fore more detailed output)
 ctest -R regression
 ```
+The gold standard files (reference solutions) used in the regression tests should automatically be downloaded during the `make` phase.
+Alternatively, you can download them by directly make the appropriate target
+```bash
+make gold_standard
+```
+or download them as a release from [GitHub](https://github.com/lanl/parthenon/releases/).
+Make sure to get the correct version matching your source
+(stored in the `REGRESSION_GOLD_STANDARD` CMake variable).
+Note: If you results are (slightly) different, that may stem from using different
+compiler/optimization options.
```

Comment:
I'm not sure it's worth expounding on this, but there's more to expound on. We shouldn't expect bitwise compatibility even if the compiler/optimization is identical if we're running on different machines. There *will* be slight differences, and this has to be something we expect and account for.

Reply:
We implicitly already account for this. The test data has been generated in double prec but stored single prec in the hdf5 files.
So far all different machine and compilers I tested successfully passed this test.
Did anyone come across a machine/configuration where our tests currently fail for those reasons?

Reply:
No. I wasn't expecting this to cause a test failure. I just meant the `README` might be misleading if you don't know about architecture-level differences.